1-js/02-first-steps/05-types/article.md

```diff
@@ -1,226 +1,250 @@
+JavaScript-ում գոյություն ոնեն 3 տեսակի չակերտներ.
 
-1. Double quotes: `"Hello"`.
-2. Single quotes: `'Hello'`.
-3. Backticks: <code>&#96;Hello&#96;</code>.
+1. Կրկնակի չակերտներ․ `"Բարև"`։
+2. Եզակի չակերտներ․ `'Բարև'`։
+3. Թեք չակերտներ․ <code>&#96;Բարև&#96;</code>։
 
-Double and single quotes are "simple" quotes. There's practically no difference between them in JavaScript.
+Կրկնակի և միակի չակերտները «պարզ» չակերտներ են։ JavaScript-ում դրանց միջև տարբերություններ չկան։
 
-Backticks are "extended functionality" quotes. They allow us to embed variables and expressions into a string by wrapping them in `${…}`, for example:
+Թեք չակերտները ունեն «ընդլայնված ֆունկցիոնալություն»։ Դրանք թույլ են տալիս ներդնել փոփոխականներ և արտահայտություններ տողի մեջ՝ դրանք 
+`${…}`-ի մեջ պարփակելու միջոցով, օրինակ․
 
 ```js run
-let name = "John";
+let name = "Դավիթ";
 
-// embed a variable
-alert( `Hello, *!*${name}*/!*!` ); // Hello, John!
+// ներդնենք փոփոխականը
+alert( `Բարև, *!*${name}*/!*!` ); // Բարև, Դավիթ!
 
-// embed an expression
-alert( `the result is *!*${1 + 2}*/!*` ); // the result is 3
+// ներդնենք արտահայտություն
+alert( `արդյունքը կլինի *!*${1 + 2}*/!*` ); // արդյունքը կլինի 3
 ```
 
-The expression inside `${…}` is evaluated and the result becomes a part of the string. We can put anything in there: a variable like `name` or an arithmetical expression like `1 + 2` or something more complex.
+`${…}`-ի մեջ ներդրված արտահայտությունը գնահատվում է, և արդյունքը դառնում է տողի մի մասը։ Մենք կարող ենք տեղադրել այնեղ ամեն ինչ՝ 
+փոփոխական (`name`) կամ թվաբանական արտահայտություն՝ `1 + 2`-ի նման կամ ավելի բարդ։
+
+Ուշադրություն դարձրեք այն փաստին, որ սա հնարավոր է միյայն թեք չակերտների դեպքում։ Մյուս չակերտները չունեն այսպիսի ներդրման 
+ֆունկցիոնալություն։
 
-Please note that this can only be done in backticks. Other quotes don't have this embedding functionality!
 ```js run
-alert( "the result is ${1 + 2}" ); // the result is ${1 + 2} (double quotes do nothing)
+alert( "արդյունքը կլինի ${1 + 2}" ); // արդյունքը կլինի ${1 + 2} (կրկնակի չակերտները ոչինչ չեն անում)
 ```
 
-We'll cover strings more thoroughly in the chapter <info:string>.
+Մենք կանրադառնանք տողերին ավելի խորացված կերպեվ <info:string> գլխում։
 
-```smart header="There is no *character* type."
-In some languages, there is a special "character" type for a single character. For example, in the C language and in Java it is called "char".
+```smart header="Գոյություն չունի *character* տիպ։"
+Որոշ լեզուներում գոյություն ունի հատուկ "character" տիպ եզակի սիմվոլների համար։ Օրինակ՝ C և Java լեզուներում այն կոչվում է «char»։
 
-In JavaScript, there is no such type. There's only one type: `string`. A string may consist of zero characters (be empty), one character or many of them.
+JavaScript-ում չկա նման տիպ։ Կա միայն մեկ տիպ՝ `string`։ string-ը կարող է լինել դատարկ, ունենալ մեկ կամ բազմաթիվ նիշեր։
 ```
 
-## Boolean (logical type)
+## Բուլյան (տրամաբանական տիպ)
```

Comment:
```suggestion
## Boolean (տրամաբանական տիպ)
```
Here and after the term `boolean` should not be translated and should be used as it.